Formula/baresip.rb

```diff
@@ -12,21 +13,36 @@ class Baresip < Formula
     sha256 high_sierra: "b99e262d153eb3414c2a6fe813be98e78f71da205d66ede0ec799d1e07f0341a"
   end
 
-  depends_on "libre"
-  depends_on "librem"
+  conflicts_with "libre", because: "both install `libre` lib"
+  conflicts_with "librem", because: "both install `librem` lib"
```

Comment:
Why are we vendoring these in the formula now?

Reply:
They have forked the projects.
https://github.com/baresip/rem/
https://github.com/baresip/re

Reply:
Does it not work with the upstream versions?

Reply:
Unfortunately, it did not work without forked versions.